dvc/command/add.py

```diff
@@ -14,49 +15,51 @@ class CmdAdd(CmdBase):
     def run(self):
         try:
             if len(self.args.targets) > 1 and self.args.file:
-                raise DvcException("can't use '--file' with multiple targets")
+                raise RecursiveAddingWhileUsingFilename()
 
-            for target in self.args.targets:
-                self.repo.add(
-                    target,
-                    recursive=self.args.recursive,
-                    no_commit=self.args.no_commit,
-                    fname=self.args.file,
-                )
+            self.repo.add(
+                self.args.targets,
+                recursive=self.args.recursive,
+                no_commit=self.args.no_commit,
+                fname=self.args.file,
+                progress=True,
+            )
 
         except DvcException:
-            logger.exception("failed to add file")
+            logger.exception("")
             return 1
         return 0
 
 
 def add_parser(subparsers, parent_parser):
-    ADD_HELP = "Take data files or directories under DVC control."
+    add_help = "Track data files or directories with DVC."
```

Comment:
This var name renaming stuff here and down below doesn't really belong to this pr :slightly_smiling_face: Also, as @jorgeorpinel noticed, if we modify these, we should also modify all the others in dvc/command/*.py, to keep the code uniformed.

Reply:
ok I'll revert the `ADD_HELP` for consistency. I'll leave the `add_parser` => `parser` as that fixes a warning and doesn't really introduce inconsistency.